provider/source/src/normalizer/mod.rs

```diff
@@ -26,7 +27,11 @@ macro_rules! normalization_provider {
                 let $toml_data: &normalizer_serde::$serde_struct =
                     self.icuexport()?.read_and_parse_toml(&format!(
                         "norm/{}/{}.toml",
-                        self.trie_type(),
+                        if $file_name == "nfd" || $file_name == "nfkd" {
```

Comment:
nit: link to issue or otherwise document why

Reply:
Added remarks and link to the comment in this PR discussion that includes bench data.